assets/javascripts/openqa.js

```diff
@@ -119,11 +119,10 @@ function renderDataSize(sizeInByte) {
 }
 
 function alignBuildLabels() {
-  var values = $.map($('.build-label'), function (el, index) {
-    return parseInt($(el).css('width'));
-  });
-  var max = Math.max.apply(null, values);
-  $('.build-label').css('min-width', max + 'px');
+  const max = Math.max(...Array.from(document.getElementsByClassName('build-label')).map(e => e.offsetWidth));
+  const style = document.createElement('style');
+  document.head.appendChild(style);
+  style.sheet.insertRule(`@media (min-width: 1000px) { .build-label { width: ${max}px; } }`);
```

Comment:
I guess you're modifying existing logic and there's a good reason to do this in javascript... just strikes me as odd that you're getting values from css to then calculate a width to set as a new rule?

Reply:
Yes, it is odd. It is what the code did before, though. The main thing I change here is introducing a media query that is in-line with the one I added on the CSS-side (so the assigned with doesn't get in the way when the layout is collapsed into one column).

Note that we could probably get rid of this JavaScript code completely by *not* using the layouting/grid of Bootstrap and using instead plain CSS. (At least I wouldn't know how to avoid it when using Bootstrap classes but I have a rough idea how to avoid it when using modern CSS.) However, that would be a greater refactoring so I'm not changing the approach as part of this PR.

Reply:
Ack. Maybe it's best anyway to see what we need for the new bs upgrade. Was just wondering if this might have been a "actually there is an easy alternative".